mxnet/buildspec.yml

```diff
@@ -12,12 +12,6 @@ repository_info:
     repository_name: &REPOSITORY_NAME !join [pr, "-", *FRAMEWORK, "-", *TRAINING_IMAGE_TYPE]
     repository: &REPOSITORY !join [ *ACCOUNT_ID, .dkr.ecr., *REGION, .amazonaws.com/, *REPOSITORY_NAME ]
 
-  inference_repository: &INFERENCE_REPOSITORY
```

Comment:
Can this file be moved into the mxnet/training folder and update the backend CI to maintain consistency for buildspecy.py file locations? the old patten is all buildspec.py  files at the framework level where the new would move the buildspec.py to the framework and image type.